pkg/controller/telemetry/accesslog.go

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package telemetry
+
+import (
+	"bufio"
+	"fmt"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+type logInfo struct {
+	direction       string
+	sourceAddress   string
+	sourceWorkload  string
+	sourceNamespace string
+
+	destinationAddress   string
+	destinationService   string
+	destinationWorkload  string
+	destinationNamespace string
+}
+
+func RunAccesslog(data requestMetric, accesslog logInfo) {
+	_ = buildAccesslog(data, accesslog)
+}
+
+func buildAccesslog(data requestMetric, accesslog logInfo) string {
+	closeTime := data.closeTime
+	startTime := getOSBootTime()
+	uptime := calculateUptime(startTime, closeTime)
```

Comment:
What confuses me is does uptime refer to the current time? Why is it calculated like this?